<launch>

  <node name="spacenav_node" pkg="spacenav_node" type="spacenav_node" />

  <node name="spacenav_to_twist" pkg="jog_arm" type="spacenav_to_twist" output="screen" />

  <node name="jog_arm_server" pkg="jog_arm" type="jog_arm_server" output="screen" >
    <param name="parameter_ns" type="string" value="jog_arm_server" />
    <rosparam command="load" file="$(find jog_arm)/config/jog_settings.yaml" />
  </node>

</launch>
